e2e/karma.conf.js

```diff
@@ -137,7 +137,7 @@ module.exports = function(config) {
 
   config.set({
     ...extraConfig,
-    browsers: ['Chrome'],
+    browsers: ['chrome_webgpu'],
```

Comment:
If keep above line unchanged, will it have errors? I think you have added corresponding flags into `bs_chrome_mac`.

Reply:
Yes,  this seems not necessary, removed